resolwe_bio/processes/alignment/hisat2.yml

```diff
@@ -58,13 +74,32 @@
 
       {% if reads|type|subtype('data:reads:fastq:single:') %}
 
-        hisat2 -p {{threads}} --no-unal -x ${INDEX} -U ${FW_READS} -S alignment_unsorted.sam 2> stats.txt
+        hisat2 \
+          -p {{threads}} \
+          --no-unal \
+          -x ${INDEX} \
+          -U ${FW_READS} \
+          -S alignment_unsorted.sam \
+          {% if noncansplice %}--pen-noncansplice {{noncansplice}}{% endif %} \
+          {% if softclip %}--no-softclip{% endif %} \
+          {% if cufflinks %}--dta-cufflinks{% endif %} \
+          2> stats.txt
         re-checkrc "HISAT2 alignment failed."
         re-progress 0.6
 
       {% else %}
 
-        hisat2 -p {{threads}} --no-unal -x ${INDEX} -1 ${FW_READS} -2 ${RW_READS} -S alignment_unsorted.sam 2> stats.txt
+        hisat2 \
+          -p {{threads}} \
```

Comment:
@jvrakor Example of how things should be changed:
https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/alignment/bowtie.yml#L295
https://github.com/genialis/resolwe-bio/blob/master/resolwe_bio/processes/alignment/bowtie.yml#L559